### ISSUE TITLE
Support TrustConfig of certificate-manager

### ISSUE THREAD

**Issue by ymmt2005** [GoogleCloudPlatform/k8s-config-connector#826]
### Checklist

- [X] I did not find a related open enhancement request.
- [X] I understand that enhancement requests filed in the GitHub repository are by default low priority.
- [X] If this request is time-sensitive, I have submitted a corresponding issue with [GCP support](https://cloud.google.com/support-hub).

### Describe the feature or resource

I wanted to manage TrustConfig of certificate-manager with Config Connector.
The recent addition of mTLS support to Cloud Load Balancing requires TrustConfig setup.
https://cloud.google.com/certificate-manager/docs/trust-configs#create

**Comment by maqiuyujoyce** [comment]
Hi @ymmt2005 , thank you for your feedback.

If it is important or time sensitive to your project, please reach out to us via [Cloud support](https://cloud.google.com/support-hub) to prioritize the request.

In addition, Config Connector resources are built on top of the corresponding Terraform resources, so we'll need to wait for Terraform implementation before we can support it in Config Connector: https://github.com/hashicorp/terraform-provider-google/issues/14984

**Comment by m2phamgg** [comment]
Terraform supports this now: https://github.com/hashicorp/terraform-provider-google/pull/15562

It would be great to have this for config connector alongside client validation field updates to `NetworkSecurityServerTLSPolicy` to enable mTLS config for load balancers.

**Comment by ldanielmadariaga** [comment]
This item has been open for >12 months. As part of scheduled repository maintenance, this Issue/PR will be closed. If this is still a critical ask, please re-open and work with your Google Cloud Account team to prioritize it with GCP KCC Product Management.

**Comment by fkc1e100** [comment]
Reopening as this resource (CertificateManagerTrustConfig) is actively requested for mTLS support by customers (b/421885405).
